modules/cluster/addons/kustomize/overlays/aws-load-balancer-controller-crds/kustomization.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
```

`bases` is deprecated.

```suggestion
resources:
```